controllers/operands/deploymentHandler.go

```diff
@@ -15,23 +15,41 @@ import (
 	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/common"
 )
 
+type deploymentOperand struct {
+	operand *genericOperand
+}
```

Comment:
This is working, but this is not what we're usually do. Why do we need the `operand` filed? usually we're doing something like:

`type deploymentOperand genericOperand`